content/en/blog/the_sandbox_launch.md

```diff
@@ -0,0 +1,54 @@
+---
+title : "The ODC Sandbox Launch"
+description: "OpenData Community Hackathon: Partners host educational Twitter Spaces"
+lead: "Fight Sybils, Decentralize Data"
+contributors: ["Stefi"]
+date: 2023-02-19T22:26:54+01:00
+lastmod: 2023-02-19T22:26:54+01:00
+draft: false
+images: []
+---
+
+Our last hackathon occurred from Jan 5th till Jan 31st , competitors collaborated to 4 different bounties, each with a different focus like helping the community growth, developing dashboards,
```

Comment:
As you may know our last hackathon took place from Jan 5th till Jan 31st and competitors could start projects in 4 different areas(bounties), each with a different focus:
- developing anti sybil- dashboards 
- hacking the hackaton by helping the community grow/ creating educational materials(such as FAQs)
- identifying sybil behaviors through exploratory data analysis 
- packaging algorithms in “Legos” that can be chained together with the intention to collaborate for an accurate classification of sybil wallets 
More details about the competition can be found in this [post](https://opendatacommunity.org/blog/data-builder-hackathon/) about the hackathon opening.